### PR TITLE
D#/kyc

### PR DESCRIPTION
**Pull request by huesforalice**

### CODE REVIEW

File: app/components/kyc/Kyc.tsx [hunk 1]
```diff
@@ -4,14 +4,89 @@ import { KycRouter } from "./Router";
...
+}
+
+interface IDispatchProps {
+  reopenRequest: () => void;
+}
+
+type IProps = IStateProps & IDispatchProps;
+
+export const KycComponent: React.SFC<IProps> = props => {
+  let requestState = props.individualRequestState;
```
Comment (krzkaczor) [inline app/components/kyc/Kyc.tsx]: could you refactor it using additional functional components and functions? The way i see it:
```
  const requestState = getCurrentRequestState(props);

  if (!requestState) {
    return <LoadingIndicator />
  }

  if (requestState.status === "draft") {
    return <KycRouter />
  }

  const status = getRequestStateStatus(props);

  return (
    <div>
      <br />
      <ProgressStepper steps={4} currentStep={4} />
      <br />
      <h1>Your KYC request</h1>
      <br />
      {status}
    </div>
  )
```

We should avoid using `let` as it makes code hard to reason about.

Comment (huesforalice) [reply app/components/kyc/Kyc.tsx]: Yeah, will do!

File: app/components/kyc/business/BusinessData.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      name="jurisdictionOfIncorporation"
+    />
+    <FormField
+      label="Legal Form"
+      touched={formikBag.touched}
+      errors={formikBag.errors}
+      name="legalForm"
+    />
+    <br /> <br />
+    <FormField
```
[the remark this answers is not in the document]
Comment (huesforalice) [reply app/components/kyc/business/BusinessData.tsx]: Sounds good!

File: app/modules/kyc/sagas.ts [hunk 3]
```diff
@@ -1,78 +1,533 @@
...
+
+function* executeLoadBeneficialOwnerFiles(boid: string): Iterator<any> {
+  try {
+    const kcyService: KycApi = yield effects.call(getDependency, symbols.apiKycService);
+    yield effects.put(actions.kyc.kycUpdateBeneficialOwnerDocuments(boid, true));
+    const result: IHttpResponse<IKycFileInfo[]> = yield kcyService.getBeneficialOwnerDocuments(
+      boid,
+    );
+    yield effects.put(actions.kyc.kycUpdateBeneficialOwnerDocuments(boid, false, result.body));
+  } catch (_e) {
```
Comment (krzkaczor) [inline app/modules/kyc/sagas.ts]: just fyi: if you dont use error you can omit it `catch {...}`

**Pull request merged**